hacep/src/main/java/org/kie/hacep/EnvConfig.java

```diff
@@ -198,6 +242,34 @@ public int getMaxSnapshotRequestAttempts() {
         return maxSnapshotRequestAttempts;
     }
 
+    public PollUnit getPollUnit() { return pollUnit; }
+
+    public PollUnit getPollSnapshotUnit() { return pollUnitSnapshot; }
+
+    public Duration getPollDuration(){
+        Duration duration;
+        if(pollUnit == PollUnit.MILLISECOND){
+            duration = Duration.ofMillis(pollTimeout);
+        }else if(pollUnit == PollUnit.SECOND){
+            duration = Duration.ofSeconds(pollTimeout);
+        }else{
+            duration = Duration.ofSeconds(1);
```

Comment:
Same comment here.
In addition we should try to refactor in a single method this logic, it is the third one with the "same" switch